src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -140,6 +143,7 @@ class Comment extends React.Component<Props, State> {
                                 <Media.Menu
```

Comment:
It looks like all `Media.Menu` components will suffer this same fate. Should we add a required `label` prop to that component and fix AppActivity and Tasks, as well?

Reply:
We could probably default it to "Options" (i18n string), since that's almost always what the "..." menu will represent